IdentityCore/src/requests/broker/MSIDSSOExtensionSilentTokenRequest.m

```diff
@@ -70,12 +72,19 @@ - (instancetype)initWithRequestParameters:(MSIDRequestParameters *)parameters
     if (self)
     {
         _tokenCache = tokenCache;
-        
+        _ssoRequestHandler = [MSIDSSORequestHandler new];
         _extensionDelegate = [MSIDSSOExtensionTokenRequestDelegate new];
         _extensionDelegate.context = parameters;
         __weak typeof(self) weakSelf = self;
         _extensionDelegate.completionBlock = ^(MSIDBrokerOperationTokenResponse *operationResponse, NSError *error)
         {
+            [weakSelf.ssoRequestHandler handleOperationResponse:operationResponse
+                                              requestParameters:weakSelf.requestParameters
+                                         tokenResponseValidator:weakSelf.tokenResponseValidator
+                                                   oauthFactory:weakSelf.oauthFactory
+                                                     tokenCache:weakSelf.tokenCache
+                                           accountMetadataCache:weakSelf.accountMetadataCache];
+            
             [weakSelf handleTokenResponse:operationResponse.tokenResponse error:error completionBlock:weakSelf.requestCompletionBlock];
```

Comment:
nit: handleTokenResponse seems very similar between silent and interactive. Might be worth combining, similar to what you did for additional token response, or with that one.

Reply:
done